corehq/apps/reports/standard/sms.py

```diff
@@ -113,9 +105,10 @@ def _fmt(val):
             )
         ]
 
-def _sms_count(user, startdate, enddate, message_type='SMSLog'):
+
+def _sms_count(user, startdate, enddate):
```

Comment:
Yeah, I've got a phone number migration planned too, I'll keep this in mind when the time comes